README.md

```diff
@@ -1,52 +1,24 @@
 401 JS -- Class 01 Lab modular patterns
 ===
```

Comment:
Remember the instructions on both Labs 1-2 tell you what to write in your README.